### DISCUSSION TITLE
Coding on Android chrome. How to  show/hide keyboard manually.

### DISCUSSION THREAD

**Discussion by frachecco86** [coder/code-server#5229] [Q&A]
HI from Rome Italy. I'm coding on my mobile browser on a remote code-server docker instance. I'm happy with the user experience so far, but I find very annoying the keyboard popping up every time I focus on the editor. I also tried Hacker Keyboard that let you hide it with volume down but still doesn't give you full control. I wish to be able to hide and show the keyboard manually. Thanks

Francesco

**Comment by jsjoeio** [comment]
Hmm...I think you'll have to figure that out at the OS level. What Android phone are you using?

**Reply by frachecco86** [reply]
I'm using Android 11 on Samsung Galaxy thanks. A22. I wonder if I should ask for a hacker keyboard mod

**Reply by jsjoeio** [reply]
Dang, I have a Pixel so I can't help much. Possibly?
